### PR TITLE
No front

### PR DESCRIPTION
**Pull request by makoto**
- Removed all front end files under `src`
- Removed frontend related npm modules
- Removed frontend related scripts under `script`
- Removed other unused files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,140 +8,45 @@
...
     "ethereum-ens": "^0.7.4",
     "ethereum-input-data-decoder": "0.0.3",
     "etherscan-api": "^3.0.11",
-    "event-emitter": "^0.3.4",
-    "file-loader": "^0.8.5",
-    "jquery": "^3.3.1",
-    "json-loader": "^0.5.4",
-    "material-ui": "^0.15.2",
-    "mathjs": "^3.2.1",
     "moment": "^2.18.1",
```
Comment (jefflau) [inline package.json:35]: Do we still need moment?

File: package.json [hunk 2]
```diff
@@ -8,140 +8,45 @@
...
-    "testEnvironment": "node",
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(js|jsx)$"
-  },
-  "babel": {
-    "presets": [
-      "react-app"
-    ]
+    "uuid": "^3.0.1"
   },
   "eslintConfig": {
```
Comment (jefflau) [inline package.json]: Do we need eslint?

Comment (jefflau) [reply package.json]: At least not the react-app part

File: package.json [hunk 3]
```diff
@@ -8,140 +8,45 @@
...
   "license": "ISC",
   "bugs": {
-    "url": "https://github.com/makoto/blockparty/issues"
+    "url": "https://github.com/noblocknoparty/blockparty/issues"
   },
-  "homepage": "https://github.com/makoto/blockparty#readme",
+  "homepage": "https://github.com/noblocknoparty/blockparty#readme",
   "dependencies": {
     "@ensdomains/ens": "^0.1.1",
     "axios": "^0.17.1",
```
Comment (jefflau) [inline package.json]: Axios is a fetch library - do we still need this?

File: package.json [hunk 4]
```diff
@@ -8,140 +8,45 @@
...
     "test": "truffle migrate && truffle test --network test",
-    "dev": "webpack-dev-server -d --inline --hot",
     "coverage": "./node_modules/.bin/solidity-coverage",
     "build-contracts": "sol-merger './contracts/*.sol' ./build",
     "solgraph": "solgraph contracts/Conference.sol  > doc/Conference.dot && dot -Tpng doc/Conference.dot -o doc/Conference.png"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/makoto/blockparty.git"
+    "url": "git+https://github.com/noblocknoparty/blockparty.git"
```
Comment (hiddentao) [inline package.json]: Please fix repo name, and in below parts too

**Review by hiddentao — COMMENTED** [review]
I like removing code. There is more we can remove but this is good for now. Please fix the repo URL in package.json

File: package.json (hunk 1 above)
Comment (makoto) [reply package.json:35]: It's used in scripts and `test/stress.js`

File: package.json (hunk 2 above)
Comment (makoto) [reply package.json]: Good catch. I will get rid of it

File: package.json (hunk 3 above)
Comment (makoto) [reply package.json]: Used it at `fetch_history.js`

File: package.json (hunk 4 above)
Comment (makoto) [reply package.json]: Good catch

**Review by hiddentao — APPROVED** [review]

**Review by jefflau — APPROVED** [review]

**Pull request merged**